Interop/Tensorflow/src/main/java/org/tribuo/interop/tensorflow/TensorFlowCheckpointModel.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2015-2021, Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.tribuo.interop.tensorflow;
+
+import org.tensorflow.exceptions.TensorFlowException;
+import org.tensorflow.proto.framework.GraphDef;
+import org.tribuo.ImmutableFeatureMap;
+import org.tribuo.ImmutableOutputInfo;
+import org.tribuo.Output;
+import org.tribuo.Prediction;
+import org.tribuo.provenance.ModelProvenance;
+import org.tensorflow.Graph;
+import org.tensorflow.Session;
+import org.tensorflow.Tensor;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * TensorFlow support is experimental, and may change without a major version bump.
+ * <p>
+ * This model encapsulates a simple model with an input feed dict,
+ * and produces a single output tensor.
+ * <p>
+ * If the checkpoint is not available on construction or after deserialisation then the
+ * model is uninitialised. Models can be initialised by calling {@link #initialize} or
+ * {@link #setCheckpointDirectory}.
+ * <p>
+ * This model's serialized form stores the weights in the specified model checkpoint directory.
+ * If you wish to convert it into a model which stores the weights inside the model then
+ * call {@link #convertToNativeModel}.
+ * <p>
+ * It accepts an {@link FeatureConverter} that converts an example's features into a {@link TensorMap}, and an
+ * {@link OutputConverter} that converts a {@link Tensor} into a {@link Prediction}.
+ * <p>
+ * The model's serialVersionUID is set to the major Tensorflow version number times 100.
+ * <p>
+ * N.B. Tensorflow support is experimental and may change without a major version bump.
```

Comment:
```suggestion
```

This is duplicated from above. In other files it is only at the start, so I removed it here.